src/app/games/connect-six/connect-six.component.ts

```diff
@@ -22,18 +24,19 @@ import { ConnectSixAlignmentMinimax } from './ConnectSixAlignmentMinimax';
 export class ConnectSixComponent extends GobanGameComponent<ConnectSixRules,
                                                             ConnectSixMove,
                                                             ConnectSixState,
-                                                            PlayerOrNone>
+                                                            PlayerOrNone,
+                                                            GobanConfig>
 {
     public droppedCoord: MGPOptional<Coord> = MGPOptional.empty();
 
     public lastMoved: Coord[] = [];
 
     public victoryCoords: Coord[] = [];
 
-    public constructor(messageDisplayer: MessageDisplayer) {
-        super(messageDisplayer);
+    public constructor(messageDisplayer: MessageDisplayer, actRoute: ActivatedRoute) {
+        super(messageDisplayer, actRoute);
         this.rules = ConnectSixRules.get();
-        this.node = this.rules.getInitialNode();
+        this.node = this.rules.getInitialNode(ConnectSixRules.DEFAULT_CONFIG);
```

Comment:
ça se passe comment le changement de config, ça change le this.node ?

Reply:
ça change le this.node, c'est fait dans getConfig.
J'ai fait un refactor, c'est comment ?

Reply:
mieux, mais à faire par défaut dans le constructeur parent plutôt !

Reply:
Sang de bite, serait-ce en train de marcher. J'allais t'envoyer le message d'erreur mais là, ça a l'air de marcher, à part deux test "new game" qui cassent.

J'avais tenté mais p-e appellé ça trop tôt.

Reply:
Mais y'a toujours un appel à `setRulesAndNode` là, il devrait pas bouger dans le constructeur général ?